examples/kotlin/example/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,34 @@
+package org.example
+
+import glide.api.GlideClient
+import glide.api.models.configuration.GlideClientConfiguration
+import glide.api.models.configuration.NodeAddress
+import java.util.concurrent.ExecutionException
+
+fun main() {
+    val host = "localhost"
+    val port = 6379
+    val useSsl = false
+
+    val config: GlideClientConfiguration =
+        GlideClientConfiguration.builder()
+            .address(NodeAddress.builder().host(host).port(port).build())
+            .useTLS(useSsl)
+            .build()
+
+    try {
+        val client: GlideClient = GlideClient.createClient(config).get()
```

Comment:
why not use kotlin's coroutines instead of calling `get`? This isn't idiomatic Kotlin.